### PR TITLE
Modified meta_curriculum JSON reading loop to only deal with .json

### PR DESCRIPTION
**Pull request by Vaspra**
See #2627
This is a bit cleaner!

Causes a `continue` in curriculum generation when the source file is not an expected JSON file.

### CODE REVIEW

**Comment by DanAmador** [comment]
Is this the reason that caused [this](https://github.com/Unity-Technologies/ml-agents/issues/2113) ?

**Comment by Vaspra** [comment]
@DanAmador Yes it is, I had not seen this issue. I was wracking my head with it last week but realised it was simply trying to json.load the Unity .meta files.

With this modification mine works fine where it previously did not.

File: ml-agents/mlagents/trainers/meta_curriculum.py [hunk 1]
```diff
@@ -30,6 +30,11 @@ def __init__(self, curriculum_folder, default_reset_parameters):
 
         try:
             for curriculum_filename in os.listdir(curriculum_folder):
+
```
Comment (chriselion) [inline ml-agents/mlagents/trainers/meta_curriculum.py]: I think black (the style checker) is complaining about whitespace on this line.

**Comment by chriselion** [comment]
As a side note, https://github.com/Unity-Technologies/ml-agents/pull/2601 hopefully made these issues easier to debug.

**Pull request merged**